README.md

```diff
@@ -1,2 +1,45 @@
 # NH Clinical
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/40c8b82dfcc74bba88c4e02770323039)](https://www.codacy.com/app/BJSS/nhclinical?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=NeovaHealth/nhclinical&amp;utm_campaign=Badge_Grade)
\ No newline at end of file
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/40c8b82dfcc74bba88c4e02770323039)](https://www.codacy.com/app/BJSS/nhclinical?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=NeovaHealth/nhclinical&amp;utm_campaign=Badge_Grade)
+
+NHClinical adds clinical models and functionality such as:
+- Clinical Activities
+- Clinical Locations
+- Clinical User Groups
+- Patients
+- Patient Admission, Discharge and Transfer
+- Patient Visits
+- Ward Staff Management
+
+These models create a basic hospital set up but it's recommended to install
+[Open-eObs](https://github.com/NeovaHealth/openeobs) which adds all the functionality
+needed to run an Acute or Mental Health hospital.
+
+## Installation
+NHClinical is a set of [Odoo](https://www.odoo.com/) modules that add clinical models 
+and functionality to Odoo.
+
+We currently develop against [our own tag of Odoo](https://github.com/bjss/odoo/tree/liveobs_1.11.1), 
+this is to ensure consistency so it's recommended when installing Odoo that you 
+install this version.
+
+Once you've downloaded Odoo, installed it's dependencies and installed PostgreSQL 9.3
+you need to update the `server.cfg` file of your Odoo installation to point to 
+the NhClinical directory.
+
+After restarting the server you can then log in as the admin user and install the
+`nh_clinical` module. This installs the different models you can work with.
+
+## Upgrading
+To upgrade the NHClinical modules you just need to update the NHClinical modules and
+press the upgrade module button for `nh_clinical`.
+
+It's recommended you backup the existing database and module files before upgrading
+so you can restore should anything go wrong.
```

Comment:
Reading this from an uninformed point of view I'd probably be thinking what's the difference between _upgrading_ and _updating_ a module and where / what is the _upgrade module button_?

I think people are either going to be familiar enough with Odoo to know how upgrading works or they won't know anything at all. These instructions assume a bit of knowledge and so are a middle-ground.

I'd say we need to explain more in-depth about what addons are, that they go in directories specified in the server config, then you have to log in as Admin, and navigate to the modules page. This long-form content would probably be more appropriate in a wiki and will also be appropriate for the **openeobs** repo.

Personally I would write it up as a wiki page in the **openeobs** repo and replace this *Upgrading* section with a recommendation that people check out the **openeobs** wiki for more information about topics such as upgrading. I don't think there's really much to say in the wiki for **nhclinical**.

Reply:
Sure will replace with Wiki page